design/overview.md

```diff
@@ -81,43 +82,36 @@ And provide a security equivalent to or greater than IBC.
 
 ## Bootstraping
 
-This is based on [Installing Peggy on a live cosmos chain](notes.md#installing-peggy-on-a-live-cosmos-chain).
-We also assume the act of upgrading the Cosmos-based binary to have peggy module is already complete,
-as approaches to that are discussed in many other places. Here we focus on the *activation* step.
+We assume the act of upgrading the Cosmos-based binary to have peggy module is already complete,
+as approaches to that are discussed in many other places. Here we focus on the _activation_ step.
 
-Set up and register orchestrators:
+1. Each `Operator` generates an Ethereum and Cosmos private key for the `Orchestrator`. These addresses are signed and submitted by the Operators valoper key. The `Orchestrator` is now free to use these delegated keys for all Peggy messages.
+1. Each `Operator` configures their `Orchestrator` with the `Peggy Bridge ID` and `start threshold` they wish, their `Orchestrator` is to reject any proposals that do not agree on these values.
```

Comment:
What is the `Peggy Bridge ID`? The ChainID of the Cosmos chain running peggy? Some ID using inside that chain to handle multiple bridges. Please define this concept, it seems important

Reply:
Ah I suppose I should define it. `Peggy Bridge ID` is just a random byte string attached to the peggy signatures to prevent collisions across multiple bridges if they where to share valdiator sets.